CONTRIBUTING.md

```diff
@@ -2,34 +2,74 @@
+Foundation (FINOS). We invite you to participate in our community by adding and
+commenting on [issues](https://github.com/finos/perspective/issues) (e.g., bug
+reports; new feature suggestions) or contributing code enhancements through a
+pull request.
 
-Note that commits and pull requests to FINOS repositories such as Perspective may only be accepted from those contributors with a [Contributor License Agreement (CLA)](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements#ContributionComplianceRequirements-ContributorLicenseAgreement) with FINOS. This may take the form of either:
-* an active, executed Individual Contributor License Agreement (ICLA) with FINOS, OR
-* coverage under an existing, active Corporate Contribution License Agreement (CCLA) executed with FINOS (most likely by the developer's employer). Please note that some, though not all, CCLAs require individuals/employees to be explicitly named on the CCLA.
+## Guidelines
 
-Commits from individuals not covered under an CLA can not be merged by Perspective's committers. We encourage you to check that you have a CLA in place well in advance of making your first pull request. 
+When submitting or commenting on an Issue, please respect the following
+guidelines. Github Issues are Perspective's project record of bugs and feature
+development, e.g. for publishing a release's Changelog, and as such it is
+important to keep them informative and on-topic. As such, please understand that
+we may remove or reclassify comments or Issues which violate the guidelines.
 
-Need an ICLA? Unsure if you are covered under an existing CCLA? Confused? Email [help@finos.org](mailto:help@finos.org) and the foundation team will help get it sorted out for you. 
+-   Be respectful and civil!
+-   Use the provided Issue templates. If the templates don't fit your need,
+    please open a [discussion](https://github.com/finos/perspective/discussions)
+    instead.
+-   Don't ask for issues to be assigned to you if you're a first-time
+    contributor. If you need help picking an issue to work on, please open a
+    [discussion](https://github.com/finos/perspective/discussions).
+-   Don't add comments asking when a feature will be delivered or a reported
+    issue fixed. The Issue will link any in-progress draft PRs or Milestones (if
+    known).
 
-If you have any general questions about contributing to Perspective, please feel free to open an issue on [github](https://github.com/finos/perspective/issues/new), or email [help@finos.org](mailto:finos.org).
+When submitting Pull Request (PR), please respect the following coding
+guidelines:
 
-## Guidelines
+-   Complete your
+    [FINOS CLA](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements#ContributionComplianceRequirements-ContributorLicenseAgreement)
+    _before_ opening a PR.
```

Comment:
I think that this PR should include a PR template that asks a user to assert that they signed the CLA before submitting.